noir-projects/noir-protocol-circuits/crates/types/src/constants.nr

```diff
@@ -117,6 +116,8 @@ pub comptime global BLOCK_MERGE_ROLLUP_INDEX: u32 = 12;
 pub comptime global ROOT_ROLLUP_INDEX: u32 = 13;
 pub comptime global BASE_PARITY_INDEX: u32 = 14;
 pub comptime global ROOT_PARITY_INDEX: u32 = 15;
+pub comptime global PRIVATE_TUBE_VK_INDEX: u32 = 16;
+pub comptime global PUBLIC_TUBE_VK_INDEX: u32 = 17;
```

Comment:
I think you'll need to regen constants - unless I'm missing something `yarn-project/noir-protocol-circuits-types/src/scripts/generate_vk_tree.ts` still has `TUBE_VK_INDEX` which no longer exists?